lib/uri.rb

```diff
@@ -44,17 +44,17 @@
 #
 # == RFC References
 #
-# A good place to view an RFC spec is http://www.ietf.org/rfc.html.
+# A good place to view an RFC spec is https://www.rfc-editor.org/.
 #
 # Here is a list of all related RFC's:
-# - RFC822[http://tools.ietf.org/html/rfc822]
-# - RFC1738[http://tools.ietf.org/html/rfc1738]
-# - RFC2255[http://tools.ietf.org/html/rfc2255]
-# - RFC2368[http://tools.ietf.org/html/rfc2368]
-# - RFC2373[http://tools.ietf.org/html/rfc2373]
-# - RFC2396[http://tools.ietf.org/html/rfc2396]
-# - RFC2732[http://tools.ietf.org/html/rfc2732]
-# - RFC3986[http://tools.ietf.org/html/rfc3986]
+# - RFC822[https://www.rfc-editor.org/rfc/rfc822]
+#  RFC1738[https://www.rfc-editor.org/rfc/rfc1738]
+#  RFC2255[https://www.rfc-editor.org/rfc/rfc2255]
+#  RFC2368[https://www.rfc-editor.org/rfc/rfc2368]
+#  RFC2373[https://www.rfc-editor.org/rfc/rfc2373]
+#  RFC2396[https://www.rfc-editor.org/rfc/rfc2396]
+#  RFC2732[https://www.rfc-editor.org/rfc/rfc2732]
+#  RFC3986[https://www.rfc-editor.org/rfc/rfc3986]
```

Comment:
Why changed this unordered list to verbatim?

```suggestion
# - RFC1738[https://www.rfc-editor.org/rfc/rfc1738]
# - RFC2255[https://www.rfc-editor.org/rfc/rfc2255]
# - RFC2368[https://www.rfc-editor.org/rfc/rfc2368]
# - RFC2373[https://www.rfc-editor.org/rfc/rfc2373]
# - RFC2396[https://www.rfc-editor.org/rfc/rfc2396]
# - RFC2732[https://www.rfc-editor.org/rfc/rfc2732]
# - RFC3986[https://www.rfc-editor.org/rfc/rfc3986]
```

Reply:
Whoops, definitely not my intention, good lookin out!